src/aat/resources/features/F-111 - Remove Case-Assigned Users and Roles/With Organisation Context/common/common/F-111_Prerequisite_Counter_Check_Call.td.json

```diff
@@ -0,0 +1,20 @@
+{
+  "_guid_": "F-111_Prerequisite_Counter_Check_Call",
+  "_extends_": "F-111_Check_Orgs_Assigned_Users_Base",
+
+  "specs": [
+    "As a prerequisite"
+  ],
+
+  "prerequisites": [
+    "F-111_Get_Organisation_Identifier_Olawale",
+    "F-111_Get_Organisation_Identifier_Hemanth",
+    "F-111_Get_Organisation_Identifier_Dil"
```

Comment:
@mattnayler - Good point. To avoid such issues, we will use real user names instead of Dil, Jamal, etc.
We have 4 solicitor users with real names and will refer to them with their actual names, in test data files.
For Dil, Jamal, etc we can setup new users with emails containing their names. Those users will be in just 1 or 2 well-known test organisations. This way, when we use users from some intended organisations, we will be able to rely on what the scenarios lines are telling us.

Reply:
Commit https://github.com/hmcts/ccd-data-store-api/pull/1045/commits/f9115e777a48fbb386ffbade9f2ed079c1e9ea31 verifies that Hemanth and Ola are in same organisation as Dil